views/update/update-beneficial-owner-type.html

```diff
@@ -50,12 +50,21 @@
   {% include "includes/back-link.html" %}
 {% endblock %}
 
+{% set hasBoMoReview = false %}
```

Comment:
To be consistent with 433, can we pull this out to the controller? There's an example on my PR